tests/nataliia_romanchuk/homework_4/test_uncompleted_comprehensions.py

```diff
@@ -0,0 +1,86 @@
+from itertools import tee
+
+# comprehensions
+some_list = [1, 2, 3, 4, 5]
+some_dict = {"a": 1, "b": 2, "c": 3}
+
+
+def test_comprehensions():
+    # generator expression
+    generator_result = list((i for i in some_list))
+    assert generator_result == some_list
+
+    # list comprehension
+    list_result = [i for i in some_list]
+    assert list_result == some_list
+
+    # set comprehension
+    set_result = {i for i in some_list}
+    assert set_result == set(some_list)
+
+    # dict comprehension
+    dict_result = {i: i * 2 for i in some_list}
+    assert dict_result == {1: 2, 2: 4, 3: 6, 4: 8, 5: 10}
+
+
+# create generator object using generator expression
+# and unpack items form it into list, set, dict, tuple
+def test_generator_expression_unpacking():
+    generator_object, generator_object1, generator_object2, generator_object3 = tee((i for i in some_list), 4)
```

Comment:
if you want to document on test do like this:

```

def test_generator_expression_unpacking():
    """
    create generator object using generator expression
    and unpack items form it into list, set, dict, tuple
    """

    generator_object, generator_object1, generator_object2, generator_object3 = tee((i for i in some_list), 4)
    ...

```

Reply:
it becomes function parameter

print(test_generator_expression_unpacking.__doc__)